agent/lib/kontena/rpc/etcd_api.rb

```diff
@@ -55,6 +55,13 @@ def delete(key, opts = {})
         {error: exc.message}
       end
 
+      # @return [Hash] {'health' => Boolean} or {'error' => String}
+      def health
+        {health: etcd.health}
+      rescue Etcd::Health::Error => error
+        {error: error.message}
```

Comment:
Not sure, I think it depends on if it's possible for the `etcd` API `GET /health` to also return `{"health": false}`?